types/convert-units/convert-units-tests.ts

```diff
@@ -0,0 +1,6 @@
+import convert from 'convert-units';
```

Comment:
Use the commonjs syntax `import convert = require('convert-units')`.

(You can still use `import convert from 'convert-units'` with `"esModuleInterop": true`.)

Reply:
Fixed